meta-resin-common/recipes-support/resin-device-uuid/resin-device-uuid/resin-device-uuid

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+help () {
+    cat << EOF
+Generate uuid for a new resin device
+resin-device-uuid [options]
+
+Options:
+    -h, --help
+        Display this help and exit.
+
+    -c, --config-path CONFIG_PATH
+        Use a non default config.json file.
+        This argument is passed to resin-vars script.
+EOF
+}
+
+# Parse arguments
+while [[ $# > 0 ]]; do
+    key=$1
+    case $key in
+        -h|--help)
+            help
+            exit 0
+            ;;
+        -c|--config-path)
+            RESIN_VARS_ARGS="$RESIN_VARS_ARGS --config-path $2"
+            shift
+            ;;
+        *)
+            echo "[WARNING] $0 : Argument '$1' unknown. Ignoring."
+            ;;
+    esac
+    shift
+done
+
+source /usr/sbin/resin-vars $RESIN_VARS_ARGS
+
+if [ -z "$CONFIG_PATH" ]; then
+    echo "[ERROR] resin-device-uuid : Please set CONFIG_PATH environment variable."
```

Comment:
That would be definitely clearer. Even though it's not a bug. If you rebase please include this suggestion too @telphan